.buildkite/x-pack/pipeline.xpack.agentbeat.yml

```diff
@@ -0,0 +1,37 @@
+env:
+  ASDF_MAGE_VERSION: 1.15.0
+  GCP_HI_PERF_MACHINE_TYPE: "c2d-highcpu-16"
+  IMAGE_UBUNTU_X86_64: "family/platform-ingest-beats-ubuntu-2204"
+  GCP_DEFAULT_MACHINE_TYPE: "c2d-highcpu-8"
```

Comment:
I guess we dont need this anymore right?

Reply:
Yeah. Let's keep it for the next PR with integration tests?

Reply:
Removed the `GCP_DEFAULT_MACHINE_TYPE`